starlight/static/sass/App.scss

```diff
@@ -0,0 +1,16 @@
+@charset 'UTF-8';
+
+// 1. Variables
+//@import "variables";
```

Comment:
These imports gave me errors

Reply:
Yes, that's normal, simply because there is no `variables.scss` file to import :)